cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,48 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
+    cy.get('#alertButton').click();
+    const stub = cy.stub();
+
+    cy.on('window:alert', stub);
 
+    // eslint-disable-next-line cypress/unsafe-to-chain-command
+    cy.get('#alertButton')
+      .click()
+      .then(() => {
+        expect(stub.getCall(0)).to.be.calledWith('You clicked a button');
```

Comment:
did you use chatGpt for this task?)

Reply:
No, i used info from this link (https://docs.cypress.io/api/cypress-api/catalog-of-events#Assert-on-the-alert-text) - Window Alert example